sharding/data_modeling.rst

```diff
@@ -10,6 +10,31 @@ Typically, you should choose that column as the distribution column which is the
 .. note::
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
-The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. In the former all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+Use Cases and Tradeoffs
+-----------------------
 
-The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregates information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
+The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. Both cases involve tradeoffs.
+
+Multi-Tenant
+~~~~~~~~~~~~
+
+* ✔ SQL Coverage
+* ✔ Ease of Migration
+* ✘ Less Parallelism
+* ✘ Less effective for large (or few) tenants
+
+In the multi-tenant use case all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+
+Existing schemas and queries typically require little adjustment during migration to the multi-tenant architecture. Additionally multi-tenant apps with few (or very large) tenants will not see a significant performance improvement.
+
+Real-Time Analytics
+~~~~~~~~~~~~~~~~~~~
+
+* ✔ Parallelism
+* ✔ Linear Scalability
+* ✘ Reduced SQL Coverage
+* ✘ Upfront Data Modeling
+
+The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregate information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
```

Comment:
Instead of 'another column', can we note something like shard by a 'lower-level entity than a tenant, for example a user-id'. 

I got it from our [blog-post](https://www.citusdata.com/blog/2016/08/10/sharding-for-a-multi-tenant-app-with-postgres/):
"It’s important to note as we did in our previous post that different distribution models can have different benefits and trade-offs. In some cases modeling on a lower level entity id such as products or purchases can be the right choice–you gain more parallelism for analytics and trade off simplicity in querying a single store. Either choice of picking a multi-tenant data model or adopt a more distributed document model can be made to scale, but each comes with its own trade-offs. If you have the need today to scale out your multi-tenant app then give Citus Cloud a try or if you have any questions on which might work best for your situation please don’t hesitate to reach out to us. We can help."